.github/PULL_REQUEST_TEMPLATE/pull_request_template_new-feature.md

```diff
@@ -0,0 +1,44 @@
+<!-- Please, fill in the description as completely as possible.-->
+
+### Description
+
+<!-- Explain the content of the new feature
+       What are the motivations? 
+       How is it integrated to the current code? -->
+
+### Testing
+
+<!-- How has this been tested?
+       What are the new test(s) and what feature/parameter does it test?
+       Are there changes and/or impacts on current tests, why?
+       How did you ensure that the solution works?
+       How will you ensure that it will continue to work in the future? -->
```

Comment:
```suggestion
<!-- How has this been tested?
       What are the new test(s) and what feature(s)/parameter(s) does it test?
       Are there changes and/or impacts on current tests, why?
       How did you ensure that the solution works?
       How will you ensure that it will continue to work in the future? -->
```